messages/culture.txt

```diff
@@ -31,4 +31,17 @@
+<b>Getting better without increasing knowledge</b> Most people think the only way to become better at their jobs is by learning some new tools or techniques that they can add to their resume. This approach is valid as long as you have a lot of time on your hands. After that, working on your thinking is going to have a far larger impact on getting better. Your ability to generalize, abstract, relate, chunk, synthesize and combine are going to help you stay relevant and competitive. So do yourself a favour and start thinking about how you think. Observe how other think in similar situations. If you need to Google for this stuff, try searching for <i>meta-cognition</i> or <i>conceptual understanding</i>.
+<b>You are responsible for your career</b> The person most responsible for your career is you. You manager is <i>also</i> responsible for your career. Your employer is <i>also</i> responsible for your career. But at the end of the day, they do not suffer if your career goes south. So when you pick tasks, try to get better. If you are handed tasks, modify your approach just enough to learn new things. If you are repeatedly working on the same thing, try it with a different toolbelt or a different process or a different mindset. If you are having trouble shaking yourself out of a rut, speak to your 1:1 manager.
+<b>Varying your work habits</b> If you feel your current work has little scope for learning, think again. Learning is not just about increasing your knowledge base. There are many other ways to vary your work and improve without actually changing your tasking. For example, you could try to increase the intensity or pace at which you do the same work. You could try to add something extra to what you already deliver. Or you could try removing a sub-task and see if your work still gets done. Perhaps you can try communicating the work you do in a different manner? Maybe reorder the sub-tasks to be a bit more creative. Alternatively, you might want to increase the quality with which you previously did something. Try some of the suggestions here and see which ones help you improve.
+<b>Increase your exposure</b> I have noticed that our group discussions are producing one point again and again. We would all benefit if we knew more :). You are going to be extremely slow if you rely on experience or classes to learn new things. You need some other way to get more exposure. My method is to read what others have to say. But reading is not for everyone. So find your own method. Maybe it is podcasts. Or videos. Perhaps it is open source contributions. Or gabbing with your ex-colleagues. Whatever it is, make sure you are exposing yourself to what is happening in other small companies and what is happening in the testing world.
```

Comment:
Instead of again and again shall we use repeatedly?

Reply:
@Annapooraniqxf2 - I have fixed this in my latest commit.